scripts/extract-l10n.sh

```diff
@@ -98,21 +100,23 @@ function create_auto_pull_request {
     echo "auto merge pull request is created ..."
 }
 
-if [ "$TRAVIS_BRANCH" != "master" ]; then
-  echo "This commit was made against the $TRAVIS_BRANCH and not the master! No extract!"
-  exit 0
-fi
-
-if [ "GITHUB_TOKEN" == "" ]
-then
-    echo "Must provide github token"
-    exit 0
-fi
-
 init_environment
 
 extract_locales
 
-commit_and_push
+git_commit
+
+if [ "GITHUB_TOKEN" = "" ]
+then
+    echo "No Github token provided, you'll have to push the branch and create the pull request yourself."
```

Comment:
I understand why we can't create the pull request without the github API access but we could still push the branch like https://github.com/mozilla/addons-frontend/pull/9375/files#diff-cae1c8c05fe95719db14844a8f236ab1R76